templates/prescription/general_list.html.twig

```diff
@@ -0,0 +1,520 @@
+                                        {{prescription.get_dosage_display()|text}} &nbsp;
+                                        </td>
+                                        {%if prescription.erx_source is empty or prescription.erx_source==0%}
+                                        <td class="editscript" id="{{prescription.id|attr}}">
+                                        {{ prescription.quantity|text }} &nbsp;
+                                        </td>
+                                        {% else %}
+                                        <td id="{{prescription.id|attr}}">
+                                        {{prescription.quantity|text}} &nbsp;
+                                        </td>
+                                        {% endif %}
+                                        <td id="{{prescription.id|attr}}">
+                                        {{prescription.get_size()|text}} {{prescription.get_unit_display()|text}}&nbsp;
+                                        </td>
+                                        <td id="{{prescription.id|attr}}">
+                                        {{prescription.refills|text}} &nbsp;
+                                        </td>
+                                        <td id="{{prescription.id|attr}}">
+                                        {{prescription.provider.get_name_display()|text}}&nbsp;
+                                        </td>
+                                        <td>
+                                            <a href="#" id="deleteDrug" class="btn btn-danger btn-sm btn-delete" data-prescription-id="{{ prescription.id|attr }}">{{'Delete'|xlt}}</a>
+                                        </td>
+                                    </tr>
+                                    {% endfor %}
+                                </tbody>
+                            </table>
+                        </div>
+                    </form>
+                </div>
+            </div>
+            {% if rx_show_drug_drug %}
+                <div id="drug-drug">
+                    <hr>
+                    <h3>{{'Drug-Drug Interaction'|xlt}}</h3>
+                    <p title="{{'Severity information will be missing if interaction is found'|xla}}"><a href="#">*{{'Notice'|xlt}}</a></p>
+                    <div id="return_info">
+                        {{INTERACTION|text}}
+                    </div>
+                    <hr>
+                </div>
+            {% endif %}
+
+            {% else %}
+            <div class="text mt-3">{{ "There are currently no prescriptions"|xlt}}.</div>
+            {% endif %}
+        </div>
+        <div class="row">
+            <div class="col-12">
+                <hr />
+            </div>
+            <div class="col-12 d-flex justify-content-end">
+                <a href="{{ CONTROLLER }}?prescription&edit&id=0&pid{{  pid|js_url }}" class="btn-primary btn-sm  mr-3" id="addButton">{{ "Add"|xlt }}</a>
```

Comment:
pid should instead be escaped by attr_url

Reply:
Fixed